src/app/project/view/issue/issue.component.html

```diff
@@ -3,9 +3,14 @@
     <div class="project-dashboard__header">
       <div class="project-dashboard__info">
         <h1 class="project-dashboard__name">{{ projectDetails.project_name }}</h1>
+
         <h3 class="project-dashboard__origin">
-          Created on <span class="text-bold">{{ projectDetails.created_on | date }}</span> by
-          <span class="text-bold">{{ projectDetails.owner }}</span>
+          <div *ngIf="projectDetails.created_on">
```

Comment:
Try to use `<span>` elements instead of DIVs when inserting just text. Since div is by default an inline-block element, it may cause positioning problems.

Reply:
@zerefwayne  Ok :)